test/aws/gather_logs.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Set path to artifact dir
+  set_fact:
+    scaleup_artifacts_dir: /tmp/artifacts/scaleup
+
+- name: Create log directory
+  local_action:
+    module: file
+    path: "{{ scaleup_artifacts_dir }}"
+    state: directory
+
+- name: Collect a list of containers
+  command: crictl ps -a -q
+  register: crictl_ps_output
+  ignore_errors: true
+- name: Collect container logs
+  command: "crictl logs {{ item }}"
+  register: crictl_logs_output
+  no_log: true
+  with_items: "{{ crictl_ps_output.stdout_lines }}"
+  ignore_errors: true
+- name: Write container logs locally
+  local_action:
```

Comment:
Needs `ignore_errors: true`, so that at least journals could be stored

Reply:
Done. Thanks!